### PR TITLE
fix(appmain): Fix multiple creations of mailservers-not-working popup

### PR DESCRIPTION
**Pull request by borismelnik**
Closes: #5899

### What does the PR do

Fix multiple creations of mailservers-not-working popup

### Affected areas

mailservers-not-working popup

### CODE REVIEW

File: ui/app/mainui/AppMain.qml [hunk 1]
```diff
@@ -73,12 +73,12 @@ Item {
...
-                    downloadURL: newVersionJSON.url,
-                    currentVersion: rootStore.profileSectionStore.getCurrentVersion(),
-                    newVersion: newVersionJSON.version
-                })
+                                                                    {
+                                                                        newVersionAvailable: newVersionJSON.available,
+                                                                        downloadURL: newVersionJSON.url,
+                                                                        currentVersion: rootStore.profileSectionStore.getCurrentVersion(),
+                                                                        newVersion: newVersionJSON.version
+                                                                    })
```
Comment (PascalPrecht) [inline ui/app/mainui/AppMain.qml:81]: This looks off. Is this intended?

**Review by PascalPrecht — CHANGES_REQUESTED** [review]
A lot of the changes here distract from the actual changes of this PR.
Can you update it to not format all the files?

File: ui/app/mainui/AppMain.qml (hunk 1 above)
Comment (borismelnik) [reply ui/app/mainui/AppMain.qml:81]: like CMD+S I automatically use CMD+A/CMD+I which call qml formatting :) I will fix

**Review by PascalPrecht — APPROVED** [review]
Much better - thanks!

**Review by elina2015 — APPROVED** [review]
Tested and Approved

**Pull request merged**